torch/csrc/jit/fuser/executor.cpp

```diff
@@ -97,25 +97,37 @@ static c10::optional<std::vector<int64_t>> canRunKernel(
 // (see above).
 // Note: Arguments are mutated by this call, although map_size is restored
 // to its original value.
-static void expandArgs(
+static bool expandArgs(
     const KernelSpec& spec,
     std::vector<at::Tensor>& args,
-    std::vector<int64_t>& map_size) {
+    std::vector<int64_t>& map_size, bool should_expand) {
```

Comment:
nit: maybe `dry_run` is a better name for `should_expand`, it's a little weird hearing "should_expand" for a function "expandArgs" that implies it always expands its arguments.